src/MeetingsSDKAdapter/controls/SwitchSpeakerControl.test.js

```diff
@@ -0,0 +1,69 @@
+import {first} from 'rxjs/operators';
+import MeetingsSDKAdapter from '../../MeetingsSDKAdapter';
+import createMockSDK from '../../mockSdk';
+
+describe('Switch Microphone Control', () => {
+  let meeting;
+  let meetingID;
+  let meetingsSDKAdapter;
+  let mockSDK;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    meetingID = 'meetingID';
+    meetingsSDKAdapter = new MeetingsSDKAdapter(mockSDK);
```

Comment:
please create a subtask for refactoring this into a function that all controls can use like this:

```js
meetingsSDKAdapter = createTestMeetingsSDKAdapter();
```

Indicative implementation:
```js
function createTestMeetingsSDKAdapter() {
  const meeting = { ... };
  const mockSDK = createMockSDK();
  const meetingsSDKAdapter = new MeetingsSDKAdapter(mockSDK);
  meetingsSDKAdapter.meetings[meetingID] = meeting;
  return meetingsSDKAdapter;
}
```

It could also be used for testing the meetings adapter itself.